rust/dialog-storage/Cargo.toml

```diff
@@ -7,6 +7,17 @@ license.workspace = true
 
 [features]
 helpers = ["tempfile", "anyhow", "rand"]
+http_tests = []
+r2_tests = []
+s3_integration_tests = []
+test-utils = [
```

Comment:
Let's pick one consistent casing for features. I have a slight preference for kebab but I could go either way.